test/ctim/specs/sighting_observed_time_test.clj

```diff
@@ -0,0 +1,39 @@
+(ns ctim.specs.sighting-observed-time-test
+  (:require [clj-momo.lib.clj-time.core :as time]
+            [clojure.spec :as s]
+            [clojure.test :refer [deftest is testing use-fixtures]]
+            [ctim.schemas.sighting :as si]
+            [ctim.test-helpers.core :as th]
+            [ctim.examples.sightings :as e]
+            [flanders.spec :as fs]))
+
+(use-fixtures :once
+  th/fixture-spec-validation
+  (th/fixture-spec si/Sighting "test.sighting"))
+
+(deftest test-sighting-observed-time-spec-validation
```

Comment:
Could you add a test to check that the observed time is valid when `start_time` is equal to `end_time`?

Reply:
Test added :)